sources/newRif/RefactorRawFiles.py

```diff
@@ -9,44 +9,43 @@
   Let's think about testing as well. For the three methods parse_pages, parse_paragraphs, prepare_output, what sort of
   inputs do they accept? Do they return the expected output?
 """
-
+import re
 from data_utilities.ParseUtil import ParsedDocument, Description, ParseState
+from sefaria.utils.talmud import section_to_daf
+from sefaria.utils.hebrew import encode_hebrew_daf
+from rif_utils import note_reg, path
 
 TEXT_OUTPUT, PARSING_STATE = [], ParseState()
 
 
 def parse_pages(doc_lines: list) -> list:
-    return []
-
+    return [''.join(page) for page in ''.join(doc_lines).split('@20')]
 
 def parse_paragraphs(page: str) -> list:
-    return []
+    new = page.replace('.', 'A').replace(':', 'B').replace('\n', 'A')
+    for note in re.findall(note_reg, new): #now return colon and period which are in refs (and for that in notes)
+        new = new.replace(note, note.replace('A', '.').replace('B', ':'))
+    new = new.replace('B', 'A')
+    return new.split('A')
 
 
 def prepare_output(segment: str, output_list: list, parsing_state: ParseState) -> None:
-    """
-    We'll use this method to get the text ready for output to a new file.
-
-    To get the current page, use PARSING_STATE.get_ref('page')
-    To get the current paragraph, use PARSING_STATE.get_ref('paragraph')
-
-    Now attach the page number to the @22 with a newline
-    Once we've edited the segment, append to TEXT_OUTPUT
-    """
+    if PARSING_STATE.get_ref('paragraph') == 1:
```

Comment:
better would be to use the variables `output_list` and `parsing_state`. When generalizing this module to run on several files, we don't want to assume that there will be one single global variable. We'll want to have the option to redefine these in each loop. This would also allow us to recycle these functions, generalize them or import them to other modules.